frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -89,15 +89,9 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 </NavDropdown>
                 )
               }
+
               {
-                !hasRole(currentUser, "ROLE_DRIVER") && isParticipant(currentUser) && (
-                  <NavDropdown title="Shifts" id="appnavbar-shift-dropdown" data-testid="appnavbar-shift-dropdown" >
-                    <NavDropdown.Item data-testid="appnavbar-shift-dropdown-shifts" as={Link} to="/shift/">Shifts</NavDropdown.Item>
-                  </NavDropdown>
-                )
-              }
-              {
-                hasRole(currentUser, "ROLE_DRIVER") && isParticipant(currentUser) && (
+               !hasRole(currentUser, "ROLE_RIDER") && isParticipant(currentUser) && (
```

Comment:
```suggestion
               hasRole(currentUser, "ROLE_DRIVER") ||    hasRole(currentUser, "ROLE_ADMIN")   (
```

I wonder if this would be more clear

Reply:
Adil: you marked this as resolved, but you didn't take the suggestion.  Do you think the current code is more clear?  Yesterday you seemed to agree with me.

Reply:
I left a comment addressing this issue in the previous requested change.